client/src/Models/CanadaNationalStats.ts

```diff
@@ -0,0 +1,36 @@
+class CanadaNationalStats {
+    CasesNew: number = 0;
+    CasesTotal: number = 0;
+    Recovered: number = 0;
+    DailyRecovered: number = 0;
+    TotalDeaths: number = 0;
+    DailyDeaths: number = 0;
+    TotalTested: number = 0;
+    DailyTested: number = 0;
+    TotalActive: number = 0;
+    DailyActive: number = 0;
+    TotalHospitalized: number = 0;
+    DailyHospitalized: number = 0;
+    TotalICU: number = 0;
+    DailyICU: number = 0;
+    TotalVaccinated: number = 0;
+    DailyVaccinated: number = 0;
+    constructor (casesNew: number, casesTotal: number, recovered: number, dailyRecovered: number, totalDeaths: number, dailyDeaths: number, totalTested: number, dailyTested: number, totalActive: number, dailyActive: number, totalHospitalized: number, dailyHospitalized: number, totalICU: number, dailyICU: number, totalVaccinated: number, dailyVaccinated: number) {
```

Comment:
This should actually be the model we use, do what was mentioned in the other file and delete the models in the backend as discussed. Just depend on the frontend for models for the time being.

Reply:
Not fully sure what you meant by this, refactored into interface with public properties like I did with the backend version.